specs/http-api/README.md

```diff
@@ -156,6 +157,17 @@ The IDs of individual tbDEX messages are used as idempotency keys
 
 ---
 
+# Callbacks
+Callbacks are implemented via the `replyTo` property on the [request to create an exchange](#create-exchange). 
+
+`replyTo` is a fully qualified URI which could either be a DID or just a URL.
+
+If `replyTo` is present, a PFI will send any/all new messages for a given exchange to the supplied URI. This makes the URI scoped to each exchange, allowing the caller to specify a different URI per exchange if they so wish. 
+
+If `replyTo` is _not_ present, the caller will have to poll the PFI for the exchange in question to receive new messages.
+
```

Comment:
just a general q about replyTo - does the URI change depending on whether Alice is talking tbdex via a self-custodial vs custodial wallet?

Reply:
The URI can actually be different per exchange. In general, no difference in terms of how the callback is used if coming from a self-custodied vs custodial wallet though.